Bottom: af9053d7d5a151d3cc54f3315ffdc06d0f0f6643
Top:    b6ddae78fb2a5554ec4ab5a119a072dc28875c21
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-12 09:06:32 -0500

Refresh of tmp-wiki-database

---
diff --git a/roles/docker-mediawiki/tasks/main.yml b/roles/docker-mediawiki/tasks/main.yml
index 9f7f65cf..4e8eb137 100644
--- a/roles/docker-mediawiki/tasks/main.yml
+++ b/roles/docker-mediawiki/tasks/main.yml
@@ -144,7 +144,7 @@
       - '{{ openssl_dv_name }}:/etc/ssl/private:z'
       - '{{ wiki_dv_name }}/var/www/html:z'
     env:
-      WIKI_HOSTNAME: '{{ container_addr_map.wiki.hostname }}'
+      WIKI_HOSTNAME: '{{ wiki_hostname }}'
       WIKI_MAIL_USER: 'wiki'
       WIKI_MAIL_PASSWORD: '{{ wiki_email_password | quote }}'
     image: '{{ wiki_image_name }}:{{ docker_image_tag }}'
